src/OrchardCore/OrchardCore.Navigation.Core/INavigationManager.cs

```diff
@@ -6,6 +6,6 @@ namespace OrchardCore.Navigation
 {
     public interface INavigationManager
     {
-        Task<IEnumerable<MenuItem>> BuildMenuAsync(string name, ActionContext context);
+        Task<IEnumerable<MenuItem>> BuildMenuAsync(string name, ActionContext context, string adminUrlPrefix);
```

Comment:
Why?

Reply:
Ignore this. I have changed the way to get it from DI.